### PR TITLE
Render cholera specific activities under their respective indicator groups [#176990423]

### PR DESCRIPTION
**Pull request by carl**
- added initial cholera crosswalk csv, rake tasks to generate json and to import the cholera actions json
- fix makeGetDisplayForDiseaseId to take an actual diseaseId
- fix display of disease name in ActionBadgePill
- add byebug for rails debugging
- updated fixture data for influenza
- new colors for cholera and ebola, and updated colors for health-system and influenza
- update system tests for happy path with influenza and cholera

### CODE REVIEW

**Review by dysnomian — APPROVED** [review]

File: app/javascript/src/components/ChartCard/BarChartLegend.js [hunk 1]
```diff
@@ -1,10 +1,9 @@
 import React from "react"
 import { useSelector } from "react-redux"
-import { getPlan, makeGetDisplayForDiseaseId } from "../../config/selectors"
+import { makeGetDisplayForDiseaseId } from "../../config/selectors"
 
 const BarChartLegend = () => {
-  const plan = useSelector((state) => getPlan(state))
-  const getDisplayForDiseaseId = makeGetDisplayForDiseaseId(plan.disease_ids)
+  const getDisplayForDiseaseId = makeGetDisplayForDiseaseId(1) // TODO: this will get cleaned up in story #
   const labelSeriesB = useSelector((state) => getDisplayForDiseaseId(state))
```
Comment (indirect) [inline app/javascript/src/components/ChartCard/BarChartLegend.js:7]: is there a missing story number here?

File: .gitignore [hunk 2]
```diff
@@ -19,3 +19,6 @@
 /public/packs-test
 /yarn-error.log
 yarn-debug.log*
+
+.idea
+.DS_Store
```
Comment (indirect) [inline .gitignore:24]: please add .idea and .DS_Store to your machine-wide gitignore. don't put your personal editor or OS decisions into a shared repo. 😄  see https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/ for more discussion

File: app/models/concerns/benchmark_indicator_action_seed.rb [hunk 3]
```diff
@@ -68,5 +69,38 @@ def seed_influenza_actions!
...
+
+    def seed_cholera_actions!
+      cholera = Disease.cholera
+      return if disease_actions_are_present?(cholera)
+
+      warn "Seeding data for Cholera Actions..."
+
+      cholera_actions_attrs =
+          JSON.parse File.read File.join Rails.root,
+                                         "/db/seed-data/cholera_actions.json"
```
Comment (indirect) [inline app/models/concerns/benchmark_indicator_action_seed.rb:81]: `Rails.root` is a `Pathname`, so you can do this: `JSON.parse Rails.root.join("db/seed-data/cholera_actions.json").read`

**Review by indirect — APPROVED** [review]
looks good! a couple minor suggestions.

File: app/javascript/src/components/ChartCard/BarChartLegend.js (hunk 1 above)
Comment (carl) [reply app/javascript/src/components/ChartCard/BarChartLegend.js:7]: fixed

File: .gitignore (hunk 2 above)
Comment (carl) [reply .gitignore:24]: fixed

File: app/models/concerns/benchmark_indicator_action_seed.rb (hunk 3 above)
Comment (carl) [reply app/models/concerns/benchmark_indicator_action_seed.rb]: fixed

**Pull request merged**